src/com/cardstore/controller/admin/ShowCatalogueServlet.java

```diff
@@ -14,11 +14,10 @@
 import jakarta.servlet.http.HttpServletResponse;
 
 /**
- * @author Sera Jeong 12211242
- * Created Date: 21/08/2024
+ * @author Sera Jeong 12211242 Created Date: 21/08/2024
  */
 
-@WebServlet("/catalogue")
+@WebServlet("/admin/catalogue")
 public class ShowCatalogueServlet extends HttpServlet {
```

Comment:
Sorry I missed this before but we need to create a CardServices class and call CardDAO from there, we should not call them directly from servlets